tests/estimators/classification/test_deeplearning_common.py

```diff
@@ -509,3 +509,94 @@ def test_learning_phase(art_warning, image_dl_estimator):
         assert hasattr(classifier, "_learning_phase")
     except ARTTestException as e:
         art_warning(e)
+
+
+# Note, the following part is for testing the amp tool on pytorch platform only.
+apex_spec = importlib.util.find_spec("apex")
```

Comment:
just out of curiosity, do you think this `amp_found` check will need to be done in other testing scripts? If yes then it would make more sense to encapsulate this code as a new pytest marker, similar in spirit to @pytest.mark.skipMlFramework defined in `conftest.py/skip_by_platform`. This is really easy to do, I can help out if it's unclear how to encapsulate it

Reply:
Solved.